exercises/robot-simulator/canonical-data.json

```diff
@@ -209,20 +207,6 @@
     {
       "description": "moves the robot forward 1 space in the direction it is pointing",
       "cases": [
-        {
-          "description": "does not change the direction",
-          "property": "advance",
-          "input": {
-            "position": {
-              "x": 0,
-              "y": 0
-            },
-            "direction": "north"
-          },
-          "expected": {
-            "direction": "north"
```

Comment:
I removed this exercise and another just like it for a couple of reasons.  First, I think they are redundant and two, I cannot infer if `x` or `y` should be changing.  It is clearly decided in other `advance` cases which variable (`x` or `y`) should be changing as the writer of the JSON made the decision.  (X for East/West and Y for North/South).